eventlet/wsgi.py

```diff
@@ -381,7 +381,12 @@ def _read_request_line(self):
         except greenio.SSL.ZeroReturnError:
             pass
         except socket.error as e:
-            if support.get_errno(e) not in BAD_SOCK:
+            erno = support.get_errno(e)
```

Comment:
Why does this line introduce error on line 389?

Reply:
Yeah, that would be my typo. Thank you for spotting that.

Reply:
Nah, typo is not the point. There will be more typos, it's fine. Problem is that tiny typo coerces to another name.

This line introduces error because name `erno` is only 1 letter distance away from `errno`. Defensive coding says make things sufficiently distinct, to avoid errors in writing and, perhaps more importantly, in reading.

If it was called `bQ8kk` or `current_errno` or something else distinctive, then it would be hard to typo into existing name even in the middle of hangover night.

Now latest patch shadows `errno` module name. It works, but name shadowing is road to trouble in general.

If you are sick from picky review, please let me know, I will change code and keep your authorship to commit history.

Thanks for your work.

Reply:
I completely agree. It's just so easy to lose track of these common sense rules when you're going for a quick patch, so thank you for taking the time to offer your insight and for reminding me.